products/storage/terraform.yaml

```diff
@@ -15,7 +15,6 @@
 overrides: !ruby/object:Overrides::ResourceOverrides
   Bucket: !ruby/object:Overrides::Terraform::ResourceOverride
     exclude_resource: true
-    error_retry_predicates: ["isStoragePreconditionError"]
```

Comment:
hii ~ 

So the rest of the change looks good but question on this. I notice in the release notes storage is quoted as having a change.

```
storage: fail fast on conflict error which would timeout without ever succeeding.
```

I could be missing something but shouldn't we be adding a `error_retry_predicates` like we do for the `filestore` resource. A little lost how removing a `error_retry_predicates` will enable us to fast fail on the 429.

Reply:
Oh, it just means we won't retry these 429s in the *inner* loop for that resource.  429s are automatically retried in the outer loop already.

I will revise that comment, though.